src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  margin-bottom: 24px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+
+  &__star {
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+    background: no-repeat url(images/star-active.svg);
+    background-position: center;
+  }
+
+  .stars__star--color_grey {
```

Comment:
Use BEM syntax for the modifier class. Change '.stars__star--color_grey' to '&--color_grey' and move it inside '&__star' block.